asapdiscovery-dataviz/asapdiscovery/dataviz/viz_targets.py

```diff
@@ -28,7 +28,7 @@
 
 # enum for allowed targets
 # TODO make configurable from YAML file
-class VizTargets(Enum):
+class VizTargets(str, Enum):
```

Comment:
(genuine question) why do this? what's the benefit? and should I be doing this in other places?

Reply:
Ah yes this is interestign. This is called an enum mixin, so allows the value of the enum to act as the type of the mixin. This is important for equality comparison. The following snippet should make it a clearer. TBH I didn't really twig on this till recently.  

```python
from enum import Enum
class normal_enum(Enum):
    a = "a"

normal_enum("a") == "a"
>>> False # compares the class instance normal_enum(a) to "a"

class str_enum(str, Enum):
    a = "a"
    
str_enum("a") == "a"
>>> True # compares as if a is str

```

Reply:
Docs [here](https://docs.python.org/3/howto/enum.html)

Reply:
oh very cool, that makes sense! thanks!